hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson

```diff
@@ -241,7 +241,9 @@ registers: [
           ''',
           resval: "3",
         },
-      ]
+      ],
+      tags: [// The status readback depends on state of ast and do not always have a set value
+        "excl:CsrAllTests:CsrExclInitCheck"]
```

Comment:
Oh one small thing to confirm: Is InitCheck exclusion enough? Do we need to make it `CsrExclCheck`?
Below are the definitions: 
```
      CsrExclInitCheck  = 3'b001, // exclude csr from init val check
      CsrExclWriteCheck = 3'b010, // exclude csr from write-read check
      CsrExclCheck      = 3'b011, // exclude csr from init or write-read check
```

Reply:
ah i didn't use `CsrExclCheck` because this register has only `ro` attribute from software, so I thought that meant a write-read check wouldn't happen anyways.

What do you think?

Reply:
The AST includes 32 registers at the moment. The last register (32nd),  when get written, will set the ast_init_done_o.

The number of registers matches the current AST NTIL accessable OT registers. This number will change once the secure code 
will be done.

Reply:
I think we will still do write after read, just to check the write value will be ignored? I think would be safer to do `CsrExclCheck`. I think it does not hurt :)

Reply:
sounds good, i will add that.